.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,60 @@
+name: Bug report 🐛
+description:
+  Something isn't working as expected? Here is the right place to report.
+title: '[YOUR TITLE]: Brief description'
+assignees:
+  - jeffchew,ljcarot,shixiedesign,RobertaJHahn
+labels: [ 'bug' ]
+body:
+  - type: markdown
+    attributes:
+      value: '## Bug report 🐛'
+  - type: markdown
+    attributes:
+      value: 'Use the form below to report any bugs you find on the Carbon for IBM.com website. Feel free to leave blank any sections that aren’t relevant but try to include as much detail as possible. The more details we have, the faster we can respond.'
+  - type: markdown
+    attributes:
+      value: 'If this is an issue for Carbon for IBM.com and not the website, please open an issue at https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/new/choose.'
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: "What did you expect to happen? What happened instead?"
```

Comment:
I suggest we add "Please attach a screenshot if relevant."

Reply:
@shixiedesign we have the question below asking for screenshots, should we move to the first question instead?

Reply:
I took a look and with the layout it works to keep the screenshot where you have it under Pages impacted. I like the division of topics with these headings. It looks easy to complete and inviting!